invalid memory address error when opening in a small window

**Describe the bug**
```
▶ gg
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x17fe936]

goroutine 1 [running]:
github.com/jesseduffield/lazygit/vendor/github.com/jesseduffield/gocui.(*View).Name(...)
	/Users/jesseduffieldduffield/go/src/github.com/jesseduffield/lazygit/vendor/github.com/jesseduffield/gocui/view.go:145
github.com/jesseduffield/lazygit/pkg/gui.(*Gui).setMainTitle(0xc0001b0400, 0x36, 0x4)
	/Users/jesseduffieldduffield/go/src/github.com/jesseduffield/lazygit/pkg/gui/context.go:24 +0x36
github.com/jesseduffield/lazygit/pkg/gui.(*Gui).onFocusChange(0xc0001b0400, 0x0, 0x0)
	/Users/jesseduffieldduffield/go/src/github.com/jesseduffield/lazygit/pkg/gui/gui.go:255 +0x56
github.com/jesseduffield/lazygit/pkg/gui.(*Gui).getFocusLayout.func1(0xc00037a000, 0x0, 0x0)
	/Users/jesseduffieldduffield/go/src/github.com/jesseduffield/lazygit/pkg/gui/gui.go:233 +0x49
github.com/jesseduffield/lazygit/vendor/github.com/jesseduffield/gocui.ManagerFunc.Layout(0xc000353ec0, 0xc00037a000, 0x0, 0x0)
	/Users/jesseduffieldduffield/go/src/github.com/jesseduffield/lazygit/vendor/github.com/jesseduffield/gocui/gui.go:353 +0x30
github.com/jesseduffield/lazygit/vendor/github.com/jesseduffield/gocui.(*Gui).flush(0xc00037a000, 0x2, 0x1)
	/Users/jesseduffieldduffield/go/src/github.com/jesseduffield/lazygit/vendor/github.com/jesseduffield/gocui/gui.go:469 +0xd2
github.com/jesseduffield/lazygit/vendor/github.com/jesseduffield/gocui.(*Gui).MainLoop(0xc00037a000, 0xc00037a000, 0x0)
	/Users/jesseduffieldduffield/go/src/github.com/jesseduffield/lazygit/vendor/github.com/jesseduffield/gocui/gui.go:377 +0x54
github.com/jesseduffield/lazygit/pkg/gui.(*Gui).Run(0xc0001b0400, 0x0, 0x0)
	/Users/jesseduffieldduffield/go/src/github.com/jesseduffield/lazygit/pkg/gui/gui.go:642 +0x367
github.com/jesseduffield/lazygit/pkg/gui.(*Gui).RunWithSubprocesses(0xc0001b0400, 0xc000235730, 0xc0000d3950)
	/Users/jesseduffieldduffield/go/src/github.com/jesseduffield/lazygit/pkg/gui/gui.go:651 +0x2f
github.com/jesseduffield/lazygit/pkg/app.(*App).Run(0xc000273730, 0xc0002356c0, 0xc000273730)
	/Users/jesseduffieldduffield/go/src/github.com/jesseduffield/lazygit/pkg/app/app.go:161 +0x6b
main.main()
	/Users/jesseduffieldduffield/go/src/github.com/jesseduffield/lazygit/main.go:51 +0x428
```

**To Reproduce**
open lazygit in a really small window

**Expected behavior**
it opens

**Desktop (please complete the following information):**
 - OS: OSX
 - Lazygit Version master 3e779bc

**Additional context**
This may be a quick win, I haven't looked into the stack trace

--- a/pkg/gui/context.go
+++ b/pkg/gui/context.go
@@ -21,7 +21,11 @@ func (gui *Gui) contextTitleMap() map[string]map[string]string {
 }
 
 func (gui *Gui) setMainTitle() error {
-	currentViewName := gui.g.CurrentView().Name()
+	currentView := gui.g.CurrentView()
+	if currentView == nil {
+		return nil
+	}
+	currentViewName := currentView.Name()
 	var newTitle string
 	if context, ok := gui.State.Contexts[currentViewName]; ok {
 		newTitle = gui.contextTitleMap()[currentViewName][context]

--- a/pkg/gui/files_panel.go
+++ b/pkg/gui/files_panel.go
@@ -86,6 +86,10 @@ func (gui *Gui) refreshFiles() error {
 	selectedFile, _ := gui.getSelectedFile(gui.g)
 
 	filesView := gui.getFilesView()
+	if filesView == nil {
+		// if the filesView hasn't been instantiated yet we just return
+		return nil
+	}
 	if err := gui.refreshStateFiles(); err != nil {
 		return err
 	}

